Fitter/scripts/plot_maker.C

```diff
@@ -2053,7 +2053,7 @@ void plot_maker(std::string postfix = "") {
     std::string out_dir = "/afs/crc.nd.edu/user/f/fyan2/macrotesting/CMSSW_10_2_13/src/EFTFit/Fitter/test/fit_results/";
     
     //TString fpath_datacard = "/afs/crc.nd.edu/user/f/fyan2/macrotesting/CMSSW_10_2_13/src/EFTFit/Fitter/test/card_ub/combinedcard.txt";
-    TString fpath_datacard = "/afs/crc.nd.edu/user/f/fyan2/macrotesting/CMSSW_10_2_13/src/EFTFit/Fitter/test/card_anatest25/combinedcard.txt";
+    TString fpath_datacard = "/afs/crc.nd.edu/user/f/fyan2/macrotesting/CMSSW_10_2_13/src/EFTFit/Fitter/test/card_anatest24/combinedcard.txt";

```

Comment:
Can you change this back to 25?

Reply:
I can. But isn't anatest24 the version being reviewed right now? I think that's also the frozen version.

Reply:
The version anatest24 is what is currently public (it's what's in the PAS), but the latest version is anatest25 which has the 4t xsec update (and this is what's in the paper and what will eventually be published)

Reply:
Fair enough. We can change it later when we switch to that version.

Reply:
Sorry for not clarifying this above: @bryates I agree with you, I think it would be good to switch this to anatest25 since that's the latest version, and I think (or at least I hope...) we will not have to use anatest24 for anything ever again, since the PAS is public and out the door. 

So @yanfr0818 if this line is used for determining what version of the analysis is most current, then I would completely agree with @bryates that it would be good to list that as anatest25.

Reply:
OK. I'll change it back to anatest25 in the next commit.